.github/workflows/claude.yml

```
@@ -1,29 +1,34 @@
-name: Claude Code
+name: Claude Code (Enhanced)
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)